ksqldb-rest-app/src/test/java/io/confluent/ksql/api/AuthTest.java

```diff
@@ -415,6 +436,22 @@ private void shouldFailInsertRequest(final String username, final String passwor
     validateError(expectedErrorCode, expectedMessage, insertsResponse.error);
   }
 
+  // auth header is omitted if username and password are null
+  private static HttpResponse<Buffer> sendGetRequestWithCreds(
+      final WebClient client,
+      final String uri,
+      final String username,
+      final String password
+  ) throws Exception {
+    VertxCompletableFuture<HttpResponse<Buffer>> requestFuture = new VertxCompletableFuture<>();
+    HttpRequest<Buffer> request = client.get(uri);
+    if (username != null || password != null) {
+      request = request.basicAuthentication(username, password);
+    }
+    request.send(requestFuture);
+    return requestFuture.get();
+  }
+
   private HttpResponse<Buffer> sendRequestWithCreds(
```

Comment:
nit: can we rename this to `sendPostRequestWithCreds()` in light of the new method above?

Reply:
Done